NGSTACK-636 implement form theme for bold media site forms

petarjakopec opened the pull request.

emodric reviewed (commented):
I think that information_collection_theme.html twig is wrong here. It is used in forms which are not info collectors. Whats wrong with just theme.html.twig?

petarjakopec commented:
@emodric theme naming fixed in d961a765f88a57e92d618324fa02bafdf12e2a08

pspanja commented:
Looks good to me, @Ljudevit ?

Ljudevit commented:
I believe it is ready for testing. thank you @petarjakopec !

pspanja commented on templates/themes/app/forms/theme.html.twig:
```diff
@@ -1,17 +1,21 @@
+    {% if compound %}
+        {{- form_widget(form) -}}
+    {% else %}
+        <div class="form-group {% if errors|length > 0 %}error-input{% endif %}">
```
I think we should have this wrapper by default, and kill it only for the hidden fields.

petarjakopec commented:
@pspanja @Ljudevit I added two more commits. In be27502a16cf2d94203b09fcdb64f271888b6bf6 I exposed hidden types in form theme. 
In 9fcaf1e7ce2e3e1bb51bf1f05cce2af782cc3492 I added explicit form_row rendering for newsletter and gdpr checkboxes.
